api/api.go

```diff
@@ -642,14 +642,15 @@ func (c *client) executeWithRetryAuthenticate(ctx context.Context, method, uri s
 	}
 	// check if we need to Re-authenticate
 	if e, ok := err.(*JSONError); ok {
-		log.Debug(ctx, "Error in response. Method:%s URI:%s Error: %v JSON Error: %+v", method, uri, err, e)
 		if e.StatusCode == 401 {
-			log.Debug(ctx, "need to re-authenticate")
+			log.Debug(ctx, "Authentication expired. Trying to re-authenticate")
```

Comment:
can you update the msg to "Authentication failed..." from "Authentication expired... "